<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZeroSensorPage</class>
 <widget class="QWidget" name="ZeroSensorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>473</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="firstStepLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1. Держите наконечник
датчика влажным в течение
всего процесса обнуления
(Смотрите инструкцию к датчику)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSvgWidget" name="guideSVG" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="secondStepLabel">
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2. Держите датчик неподвижно,
затем нажмите обнулить</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="AnimatedToolButton" name="resetSensorButton"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="readyOrNotLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Датчик не готов к обнулению</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedToolButton</class>
   <extends>QToolButton</extends>
   <header>animatedtoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header>qsvgwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
